### PR TITLE
Add Addiko Bank

### PR DESCRIPTION
**Pull request by anotherus3r**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -6,6 +6,17 @@ websites:
       img: actorsfcu.png
       tfa: No
 
+    - name: Addiko Bank
+      url: https://www.addiko.hr/
+      img: addikobank.png
+      tfa: Yes
+      sms: No
```
Comment (Jawshy) [inline _data/banking.yml]: `sms: No` can be removed because it should only be used when set to `Yes`.

File: _data/banking.yml [hunk 2]
```diff
@@ -6,6 +6,17 @@ websites:
...
+      phone: No
```
Comment (Jawshy) [inline _data/banking.yml]: [As above.](https://github.com/2factorauth/twofactorauth/pull/2410/files#r103431716)

File: _data/banking.yml [hunk 3]
```diff
@@ -6,6 +6,17 @@ websites:
...
+      hardware: Yes
+      software: Yes
+      doc: https://www.addiko.hr/sites/addiko.hr/files/content/addiko_bank_uputa_za_koristenje_tokena_mtokena.pdf
+      lang: hr
```
Comment (Jawshy) [inline _data/banking.yml]: `lang: hr` can be removed because it should not be used when `tfa` is set to `Yes`.

**Review by Jawshy — DISMISSED** [review]
Thanks for another pull request, @randomeuropean. 

Could you please correct the errors listed [here](https://travis-ci.org/2factorauth/twofactorauth/builds/206129446)?

**Review by stephengroat — APPROVED** [review]
my Hungarian is non-existent, but i can confirm that the formatting is correct!

**Review by fpigerre — APPROVED** [review]
@stephengroat It's all good that your Hungarian is non-existent, because the bank is Croatian 🇭🇷  😝

**Comment by fpigerre** [comment]
Thanks for the pull request @randomeuropean 👍

**Pull request merged**